lib/tests/streamlit/elements/audio_test.py

```diff
@@ -22,17 +22,17 @@
 from scipy.io import wavfile
 
 import streamlit as st
-from streamlit.runtime.media_file_manager import (
+from streamlit.runtime.media_file_storage import MediaFileStorageError
+from streamlit.runtime.memory_media_file_storage import (
     _calculate_file_id,
-    media_file_manager,
-    STATIC_MEDIA_ENDPOINT,
 )
+from streamlit.web.server.server import MEDIA_ENDPOINT
 from tests import testutil
 
 
 class AudioTest(testutil.DeltaGeneratorTestCase):
-    def test_st_audio(self):
-        """Test st.audio."""
```

Comment:
This was a monolithic "test all the things" test function that I split into smaller units.